docs/sop/PRODUCTION_TROUBLESHOOT.md

```diff
@@ -0,0 +1,275 @@
+# Troubleshooting guide for the GigaDB Website deployment
```

Comment:
**question**: Are we still using portainer? If so, can instructions for using be added into this document?

Reply:
@pli888, @kencho51 

**suggestion**: Yes, we are, and the instruction is a URL and login/password saved in GitLab variables. And I agree it should be mentioned here

* https://portainer.staging.gigadb.org/
* https://portainer.beta.gigadb.org/

Reply:
Hi,

After the live deployment, the live url will be updated to `gigadb.org`, so the portainer url should be `https://portainer.gigadb.org` which I would use in the troubleshooting guide.